### DISCUSSION TITLE
Search nodes not connecting to manager after rebuilding one

### DISCUSSION THREAD

**Discussion by jschult** [Security-Onion-Solutions/securityonion#3861] [Unsupported Versions]
Setup: 
Version 2.3.40 / Installed from ISO
1 - Management 
4 - Search
3 - Forward

Issue:
Had a drive fail on one of the search nodes and it was using RAID 0. After reconfiguring that search node with the same hostname and IP, it came back up fine, but now the other three search nodes are not connected to the manager when looking at 'GET _remote/info' in Kibana.

I've tried restarting one of the effected search nodes, as well as the manager, but no luck. The indices are still growing on the search nodes, the data is just not available in Kibana. 

I am able to ssh and curl port 9200 on each of the search nodes from the manager. 
'so-status' shows OK on the manager and search nodes. 
'salt-key' shows everything as accepted. 
'salt-call state.highstate' doesn't show any failures.
All search nodes are still showing in the firewall under hostgroups.local.yaml on the manager.
Looked at the docker network interfaces and they are still showing traffic.
Can't see anything related in the elasticsearch logs on the search nodes or the manager. 

Any thoughts or guidance would be greatly appreciated!

**Comment by jschult** [comment]
Ended up manually adding search node IPs in Stack Management > Remote Clusters. API call is '_cluster/settings'. 
Seems like DNS wasn't resolving even though curl from the manager to the search node works fine on 9200 with the hostname. 
Anyone have any ideas of why this would be the case?

**Reply by jschult** [reply]
Had logstash die without errors today on the three search nodes that previously had connection issues. After 'so-logstash-restart' on those nodes, the redis queue came back down on the manager. Still trying to figure out why things are randomly having issues.

**Reply by defensivedepth** [reply]
So no errors in `/opt/so/log/logstash.log` ?

May need to up the logging output to debug.
